.github/workflows/test.yaml

```diff
@@ -37,7 +37,7 @@ jobs:
         # specified below, these parameters have no meaning on their own and
         # gain meaning on how job steps use them.
         jupyterlab_version: [2, 3]
-        python: [3.6, 3.7, 3.8, 3.9]
+        python: ["3.6", "3.7", "3.10", "3.11"]
```

Comment:
3.6 is EOL, probably just drop that, too. Might just need `3.7` and `3.11`.

Reply:
I agree deal! I'd like to make a dedicated PR for dropping support for python 3.6 though.